src/app/scopes.ts

```diff
@@ -1,6 +1,20 @@
 import { PermissionScope } from "./base";
 
 export const PermissionScopes: PermissionScope[] = [
+  {
+    name: "AccessReview.Read.All",
+    description: "Read all access reviews that user can access",
```

Comment:
Add missing indefinite article to the description: "Read all access reviews that a user can access"

Reply:
Fixed.